models/_sector/erc4337/userops/arbitrum/_schema.yml

```diff
@@ -0,0 +1,121 @@
+version: 2
+
+models:
+
+  - name: erc4337_v0_5_arbitrum_userops_basics
+    meta:
+      blockchain: arbitrum
+      project: erc4337
+      contributors: 0xbitfly, hosuke
+    config:
+      tags: ["arbitrum", "erc4337", "account", "abstraction", "entrypoint"]
+    description: erc4337 v0.5 user operations on arbitrum
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - userop_hash
+            - tx_hash
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "blockchain which EntryPoint was deployed"
+      - &version
+        name: version
+        description: "version of the EntryPoint contract"
+      - &block_time
+        name: block_time
+        description: "UTC event block time of each User Operation"
+      - &entrypoint_contract
+        name: entrypoint_contract
+        description: "the contract address of the EntryPoint"
+      - &tx_hash
+        name: tx_hash
+        description: "Unique transaction hash value tied to each transaction on the User Operation"
+      - &sender
+        name: sender
+        description: "the sender(AA wallet address) that submmited the user operation"
+      - &userop_hash
+        name: userop_hash
+        description: "Unique user operation hash"
+      - &success
+        name: success
+        description: "weather the user operation is successfully execuated"
+      - &paymaster
+        name: paymaster
+        description: "the address that pay for the user operation gas fees on behalf of the user"
+      - &op_fee
+        name: op_fee
+        description: "The user operation gas paid by sender or paymaster. It’s depends on the chain gas setting."
+      - &beneficiary
+        name: beneficiary
+        description: "the address where the bundler setting for the user operation pay for thier gas fee"
+
+  - name: erc4337_v0_6_arbitrum_userops_basics
+    meta:
+      blockchain: arbitrum
+      project: erc4337
+      contributors: 0xbitfly, hosuke
+    config:
+      tags: ["arbitrum", "erc4337", "account", "abstraction", "entrypoint"]
+    description: erc4337 v0.6 user operations on arbitrum
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - userop_hash
+            - tx_hash
+    columns:
+      - *blockchain
+      - *version
+      - *block_time
+      - *entrypoint_contract
+      - *tx_hash
+      - *sender
+      - *userop_hash
+      - *success
+      - *paymaster
+      - *op_fee
+      - *beneficiary
+
+  - name: erc4337_arbitrum_userops_beta
```

Comment:
Seems the framework is working.
[Sample erc4337_arbitrum_userops_beta](https://dune.com/queries/2631954?d=1)